holidays/countries/laos.py

```diff
@@ -0,0 +1,452 @@
+        # 15th Waxing Day of Month 12.
+
+        # Boun That Luang Festival.
+        self._add_loy_krathong(tr("ວັນບຸນທາດຫລວງ"))
+
+        # ວັນຄູແຫ່ງຊາດ
+        # Status: In-Use.
+        # In recognition of First Lao Teacher, Kham, as started in Oct 7, 1994.
+
+        if self._year >= 1994:
+            # National Teacher Day.
+            self._add_holiday_oct_7(tr("ວັນຄູແຫ່ງຊາດ"))
+
+    def _populate_workday_holidays(self):
+        # No Public Holidays are issued, though still observed by the government.
+
+        # Available post-Lao PDR proclamation on Dec 2, 1975.
+        if self._year <= 1975:
+            return None
+
+        # ວັນສ້າງຕັ້ງກອງທັບປະຊາຊົນລາວ
+        # Status: In-Use.
+        # Celebrated the creation of the independent Lao army on Jan 20, 1949.
+
+        # Lao People's Armed Force Day.
+        self._add_holiday_jan_20(tr("ວັນສ້າງຕັ້ງກອງທັບປະຊາຊົນລາວ"))
+
+        # ວັນສ້າງຕັ້ງສະຫະພັນກໍາມະບານລາວ
+        # Status: In-Use.
+        # Celebrated the creation of Lao Federation of Trade Unions on Feb 1, 1966.
+
+        # Lao Federation of Trade Union's Day.
+        self._add_holiday_feb_1(tr("ວັນສ້າງຕັ້ງສະຫະພັນກໍາມະບານລາວ"))
+
+        # ວັນສ້າງຕັ້ງພັກປະຊາຊົນປະຕິວັດລາວ
+        # Status: In-Use.
+        # Celebrated the creation of the Lao People's Revolutionary Party on Mar 22, 1955.
+
+        # Establishment Day of the Lao People's Revolutionary Party.
+        self._add_holiday_mar_22(tr("ວັນສ້າງຕັ້ງພັກປະຊາຊົນປະຕິວັດລາວ"))
+
+        # ວັນສ້າງຕັ້ງສູນກາງຊາວໜຸ່ມປະຊາຊົນປະຕິວັດລາວ
+        # Status: In-Use.
+        # Celebrated the creation of the Lao People's Revolutionary Youth Union on Apr 14, 1955.
+
+        # Lao People's Revolutionary Youth Union Day.
+        self._add_holiday_apr_14(tr("ວັນສ້າງຕັ້ງສູນກາງຊາວໜຸ່ມປະຊາຊົນປະຕິວັດລາວ"))
+
+        # ວັນເດັກສາກົນ (`WORKDAY`)
+        # Status: Defunct, Still Observed.
+        # Starts as public holiday after Lao PDR joined UN Convention on the
+        # Rights of the Child in 1989 (de-facto start as holiday in 1990).
+        # Became defunct from 2018 onwards. Still accessible in `WORKDAY` category.
+
+        if self._year >= 2018:
+            # International Children Day.
+            self._add_childrens_day(tr("ວັນເດັກສາກົນ"))
+
+        # ວັນປູກຕົ້ນໄມ້ແຫ່ງຊາດ
+        # Status: In-Use.
+        # No information on when this was first observed is available in Thai or English sources.
+
+        # National Arbor Day.
+        self._add_holiday_jun_1(tr("ວັນປູກຕົ້ນໄມ້ແຫ່ງຊາດ"))
+
+        # ວັນຄ້າຍວັນເກີດ ທ່ານ ປະທານ ສຸພານຸວົງ
+        # Status: In-Use.
+        # Celebrated President Souphanouvong's Birthday Anniversary on Jul 13, 1909.
+
+        # President Souphanouvong's Birthday.
+        self._add_holiday_jul_13(tr("ວັນຄ້າຍວັນເກີດ ທ່ານ ປະທານ ສຸພານຸວົງ"))
+
+        # ວັນປ່ອຍປາ ແລະ ວັນອະນຸລັກສັດນ້ຳ-ສັດປ່າແຫ່ງຊາດ
+        # Status: In-Use.
+        # No information on when this was first observed is available in Thai or English sources.
```

Comment:
[Here](https://jclao.com/laos-national-day-for-wildlife-conservation-shows-its-dedication-to-animals/) says it started in 1997.

Reply:
Good catch!

Reply:
Thanks to that source, I've come across the [starting date](https://www.iufro.org/download/file/7380/5120/Lao_PDR_pdf/) for `National Arbor Day` too; now all holidays have their proper origin date :)